src/TwigComponent/doc/index.rst

```diff
@@ -56,18 +58,28 @@ Let's get this thing installed! Run:
 
     $ composer require symfony/ux-twig-component
 
-That's it! We're ready to go!
+That's it! We're ready to go! If you're not using Symfony Flex, add a config
+file to control the template directory for your components:
+
+.. _default_config:
 
-Creating a Basic Component
---------------------------
+```
+# config/packages/twig_component.yaml
+twig_component:
+    defaults:
+        # default component template directory
+        App\Twig\Components\: 'components/'
+```
```

Comment:
I used the totally wrong syntax here. And, we need to add the anonymous components config